actions/storage.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/spf13/afero"
 )
 
-func newStorage() (storage.Storage, error) {
+func newStorage() (storage.Backend, error) {
```

Comment:
minor: i understand that linter will complain about package and interface name being the same, but backend does not make sense to me. Client, GetterSaver, Provider non of them are good tbh but i don;t like backend for some reason

Reply:
I wasn't sure myself. Service maybe? but it's very general...

Reply:
what about driver? makes sense?

Reply:
Could we move this decision to a follow-up?